### PR TITLE
Bind NOTIFICATION_LOCAL_TRANFORM_CHANGED

### PR DESCRIPTION
**Pull request by jasonwinterpixel**
The NOTIFICATION_LOCAL_TRANFORM_CHANGED constant is not exposed to GDScript, which means that using `canvas_item.set_notify_local_transform` requires hard coding '35' into your GDScript.

This PR binds this constant to GDScript.

3.x version https://github.com/godotengine/godot/pull/59353

### CODE REVIEW

File: doc/classes/CanvasItem.xml [hunk 1]
```diff
@@ -439,14 +439,14 @@
...
-				If [code]enable[/code] is [code]true[/code], children will be updated with local transform data.
+				If [code]enable[/code] is [code]true[/code], this node will receive [constant NOTIFICATION_LOCAL_TRANSFORM_CHANGED] when it's local transform changes.
 			</description>
 		</method>
 		<method name="set_notify_transform">
 			<return type="void" />
 			<argument index="0" name="enable" type="bool" />
 			<description>
-				If [code]enable[/code] is [code]true[/code], children will be updated with global transform data.
+				If [code]enable[/code] is [code]true[/code], this node will receive [constant NOTIFICATION_TRANSFORM_CHANGED] when it's global transform changes.
```
Comment (Calinou) [inline doc/classes/CanvasItem.xml:449]:
```suggestion
				If [code]enable[/code] is [code]true[/code], this node will receive [constant NOTIFICATION_TRANSFORM_CHANGED] when its global transform changes.
```

File: doc/classes/CanvasItem.xml [hunk 2]
```diff
@@ -439,14 +439,14 @@
 			<return type="void" />
 			<argument index="0" name="enable" type="bool" />
 			<description>
-				If [code]enable[/code] is [code]true[/code], children will be updated with local transform data.
+				If [code]enable[/code] is [code]true[/code], this node will receive [constant NOTIFICATION_LOCAL_TRANSFORM_CHANGED] when it's local transform changes.
```
Comment (Calinou) [inline doc/classes/CanvasItem.xml:442]: Same as in the `3.x` PR:

```suggestion
				If [code]enable[/code] is [code]true[/code], this node will receive [constant NOTIFICATION_LOCAL_TRANSFORM_CHANGED] when its local transform changes.
```

**Review by Calinou — DISMISSED** [review]

**Comment by jasonwinterpixel** [comment]
Fixed

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**